src/main/java/org/tron/program/Benchmark.java

```diff
@@ -0,0 +1,378 @@
+
+    long totalDuration = 0;
+    int repeatCount = 12;
+
+    for (int i = 0; i < repeatCount; i++) {
+      long curDuration = triggerContractAndReturnDuration(contractAddress, feeLimit);
+      // System.out.println(String.format("count: %d, duration: %d", i, curDuration));
+      if (i >= 2) {
+        totalDuration += curDuration;
+      }
+      Thread.sleep(10);
+    }
+    long avgDuration = totalDuration / (repeatCount - 2);
+    // System.out.println(String.format("avg duration: %d", avgDuration));
+
+    double defaultMinTimeRatio = 0.0;
+    double defaultMaxTimeRatio = 5.0;
+    long timeBenchmark = 50L;
+    return new TimeBenchmarkResult(defaultMinTimeRatio * avgDuration / timeBenchmark,
+        defaultMaxTimeRatio * avgDuration / timeBenchmark);
+
+  }
+
+  public long triggerContractAndReturnDuration(byte[] contractAddress, long feeLimit)
+      throws ContractExeException, ReceiptCheckErrException, VMIllegalException, ContractValidateException {
+    /* ====================================================================== */
+    String params = "0000000000000000000000000000000000000000000000000000000000002fa8" +
+        "00000000000000000000000000000000000000000000000000000000000004c4" +
+        "000000000000000000000000000000000000000000000000000000000000007a";
+    byte[] triggerData = VMTimeBenchmarkUtils
+        .parseABI("timeBenchmark(uint256,uint256,uint256)", params);
+    TVMResult result = VMTimeBenchmarkUtils
+        .triggerContractAndReturnTVMResult(Hex.decode(OWNER_ADDRESS),
+            contractAddress, triggerData, 0, feeLimit, dbManager, null);
+
+    return result.getDuration();
+  }
+
+  public TimeBenchmarkResult getTimeRatio()
+      throws ContractExeException, ReceiptCheckErrException, VMIllegalException, ContractValidateException, InterruptedException {
+
+    TimeBenchmarkResult result = timeBenchmark();
+
+    return result;
+  }
+
+  public long getMem() {
+
+    com.sun.management.OperatingSystemMXBean os = (com.sun.management.OperatingSystemMXBean)
+        java.lang.management.ManagementFactory.getOperatingSystemMXBean();
+    long physicalMemorySize = os.getTotalPhysicalMemorySize();
+    System.out.println("old way memory: "+ physicalMemorySize / 1024 / 1024 +" MB");
+
+    MBeanServer mBeanServer = ManagementFactory.getPlatformMBeanServer();
+    Object attribute = null;
+    try {
+      attribute = mBeanServer.getAttribute(new ObjectName("java.lang","type","OperatingSystem"), "TotalPhysicalMemorySize");
+      long i = (Long) attribute;
+      System.out.println("new way memory: "+ i / 1024 / 1024 +" MB");
+    } catch (MBeanException e) {
+      e.printStackTrace();
+    } catch (AttributeNotFoundException e) {
+      e.printStackTrace();
+    } catch (InstanceNotFoundException e) {
+      e.printStackTrace();
+    } catch (ReflectionException e) {
+      e.printStackTrace();
+    } catch (MalformedObjectNameException e) {
+      e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1546&issues=AWXxSwHSxJR_uy9BGF1M&open=AWXxSwHSxJR_uy9BGF1M)

[ISSUE_KEY]: AWXxSwHSxJR_uy9BGF1M